lib/Sources/Tabs/TabGridModel.swift

```diff
@@ -25,7 +26,7 @@ public final class TabGridModel {
         sortedTabs.map(TabCell.tab) + [TabCell.new]
     }
 
-    var urls: [URL] { sortedTabs.map(\.url) }
+    var urls: [URL?] { sortedTabs.map(\.url) }
```

Comment:
This is going to break the storage encoding and I'm guessing will fail to save/restore?